lib/internal/util/inspect.js

```diff
@@ -652,7 +652,8 @@ function formatRaw(ctx, value, recurseTimes) {
       // The message and the stack have to be indented as well!
       if (ctx.indentationLvl !== 0) {
         const indentation = ' '.repeat(ctx.indentationLvl);
-        base = formatError(value).replace(/\n/g, `\n${indentation}`);
+        base = formatError(value, getPrefix(constructor, '', 'Error'))
+                                  .replace(/\n/g, `\n${indentation}`);
```

Comment:
I think this would look better not aligned with the second argument passed to `formatError()` as it's misleading when scanning the code visually.